install-repositories.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -e
+
+LOG_START='\n\e[1;36m' # new line + bold + color
+LOG_END='\n\e[0m' # new line + reset color
+
+printf "${LOG_START}Initializing submodules...${LOG_END}"
+
+git submodule update --init --recursive --remote --rebase --force
+
+printf "${LOG_START}Switching to 'for-allthekeeps-on-ropsten' branch...${LOG_END}"
+
+cd ./keep-subgraph
+git checkout for-allthekeeps-on-ropsten
```

Comment:
This is specific for ropsten. The script `install-repositories.sh` should either include that in the file name that it supports ropsten only, or better accept a network flag that would let to switch from `mainnet` to `ropsten`.

Reply:
This guy was handled in [this](https://github.com/keep-network/allthekeeps/pull/4/files) PR